.changeset/lazy-seahorses-share.md

```diff
@@ -0,0 +1,13 @@
+---
+"@itwin/itwinui-css": major
+---
+
+`.iui-sidenav-button.iui-expand` now requires `.iui-button .iui-field` and `data-iui-size="small"`.
+
+```diff
+  <button
+-   class="iui-button iui-button-base iui-sidenav-button iui-expand"
++   class="iui-button iui-button-base iui-field iui-sidenav-button iui-expand"
```

Comment:
nit/optional: It would be clearer if `iui-sidenav-button` is moved to the same position as in the old line.

```suggestion
-   class="iui-button iui-button-base iui-sidenav-button iui-expand"
+   class="iui-button iui-button-base iui-sidenav-button iui-field iui-expand"
```